cmd/helpers.go

```diff
@@ -39,38 +36,29 @@ func (opt *commonOptions) WithMetrics(cmd *kingpin.Application) *commonOptions {
 	return opt
 }
 
-func (opt *commonOptions) Logger() kitlog.Logger {
-	// Output logs to STDERR
-	logger := kitlog.NewLogfmtLogger(kitlog.NewSyncWriter(os.Stderr))
-
+func (opt *commonOptions) Logger() logr.Logger {
 	// While debugging, it may be useful to provide debug log lines that
 	// include sensitive or large payloads.
+	logLevel := zapcore.InfoLevel
 	if opt.Debug {
-		logger = level.NewFilter(logger, level.AllowDebug())
-	} else {
-		logger = level.NewFilter(logger, level.AllowInfo())
+		logLevel = zapcore.DebugLevel
 	}
 
+	logger := zap.New(
+		zap.Encoder(zaplogfmt.NewEncoder(zapcore.EncoderConfig{})),
+		zap.WriteTo(os.Stderr),
+		zap.Level(logLevel),
+	)
+	ctrl.SetLogger(logger)
+
 	// @template: caller is optional, but can be useful to highlight where the log-line is
 	// originating from. This configuration has to happen after we've leveled the logger, to
 	// avoid the caller being set to level.go:63 all the time.
-	logger = kitlog.With(logger, "ts", kitlog.DefaultTimestampUTC, "caller", logging.RecorderAwareCaller())
-
-	// Connect the Go standard library logger into our kitlog instance so other
-	// dependencies output via kitlog.
-	stdlog.SetOutput(kitlog.NewStdlibAdapter(logger))
-	klog.SetOutput(kitlog.NewStdlibAdapter(logger))
+	// logger = kitlog.With(logger, "ts", kitlog.DefaultTimestampUTC, "caller", logging.RecorderAwareCaller())
```

Comment:
We need to make sure we add this back in

Reply:
Added this functionality back in! 👍